Scenarios/CGCT/CGCT/PathDemoTableViewCell.m

```diff
@@ -0,0 +1,189 @@
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import <Foundation/Foundation.h>
+#import <CoreGraphics/CoreGraphics.h>
+#import <CoreText/CoreText.h>
+#import "PathDemoTableViewCell.h"
+
+@implementation PathDemoTableViewCell
+
+- (void)drawDemoIntoContext:(CGContextRef)context withFrame:(CGRect)bounds view:(UIView*)view {
+    CGContextSaveGState(context);
+
+    CGMutablePathRef light = CGPathCreateMutable();
+    CGPathMoveToPoint(light, NULL, bounds.size.width * .9, bounds.size.height * .1);
+    CGPathAddLineToPoint(light, NULL, 0, 0);
+    CGPathAddLineToPoint(light, NULL, 0, bounds.size.height);
+    CGPathAddLineToPoint(light, NULL, bounds.size.width * .9, bounds.size.height * .81);
+
+    CGContextSetRGBFillColor(context, .7, .7, 1, .5);
+    CGContextAddPath(context, light);
+    CGContextDrawPath(context, kCGPathFill);
+    CGPathRelease(light);
+
+    CGMutablePathRef windowTR = CGPathCreateMutable();
+
+    CGPoint windowsCorners[] = { CGPointMake(bounds.size.width * .9, bounds.size.height * .1),
+                                 CGPointMake(bounds.size.width * .9, bounds.size.height * .45),
+                                 CGPointMake(bounds.size.width * .6, bounds.size.height * .45),
+                                 CGPointMake(bounds.size.width * .6, bounds.size.height * .2),
+                                 CGPointMake(bounds.size.width * .9, bounds.size.height * .1) };
+
+    CGPathAddLines(windowTR, NULL, windowsCorners, 4);
+
+    CGAffineTransform windowTransform = CGAffineTransformIdentity;
+    windowTransform = CGAffineTransformScale(windowTransform, 1, -1);
+    windowTransform = CGAffineTransformTranslate(windowTransform, 0, -.91 * bounds.size.height);
+
+    CGPathRef windowBR = CGPathCreateCopyByTransformingPath(windowTR, &windowTransform);
+
+    windowTransform = CGAffineTransformTranslate(windowTransform, -bounds.size.width * .05, bounds.size.height * .136);
+    windowTransform = CGAffineTransformScale(windowTransform, .7, .7);
+
+    CGPathRef windowBL = CGPathCreateCopyByTransformingPath(windowBR, &windowTransform);
+
+    windowTransform = CGAffineTransformTranslate(windowTransform, 0, .908 * bounds.size.height);
+    windowTransform = CGAffineTransformScale(windowTransform, 1, -1);
+
+    CGPathRef windowTL = CGPathCreateCopyByTransformingPath(windowBR, &windowTransform);
+
+    CGContextSetRGBFillColor(context, 1, 1, 1, 1);
+
+    CGContextAddPath(context, windowTL);
+    CGContextAddPath(context, windowTR);
+    CGContextAddPath(context, windowBL);
+    CGContextAddPath(context, windowBR);
+    CGContextSetShadow(context, CGSizeMake(10.f, 10.f), 1.0);
+    CGContextDrawPath(context, kCGPathFill);
+    CGContextRestoreGState(context);
+    CGPathRelease(windowTR);
```

Comment:
External applications may be able to consume smarttypes; we might like to do that going forward